### DISCUSSION TITLE
Tab labels invisible

### DISCUSSION THREAD

**Discussion by hrrrps** [sumatrapdfreader/sumatrapdf#3786] [General]
I am running .3.5.15741 64 bit pre-release on Windows 11

Tabs at the top of the window are not showing the names of documents - i.e. no text labeling of individual tabs

**Comment by gerryqd** [comment]
I met similar thing too. 
Seems the label text is normal at first, then after I opened serval documents, add annotation to document, and tried to save the document with annotations, then the label text of all tabs disappeared.

**Comment by GitHubRulesOK** [comment]
@hrrrps 
Please try to illustrate problem with a screen shot in your initial comment but sounds like  new issue https://github.com/sumatrapdfreader/sumatrapdf/issues/3789

@gerryqd 
That sounds like a different issue in that when FOCUS of mouse activity is in another modal dialog such as help about OR edit annotations or printing etc. then tooltips are outside of that MAIN tabs framework so not activated.

**Comment by mini-rooter** [comment]
Hi all, experiencing the same issue after updating.

love the update as now I no longer accidentally move annotations around looking at PDFs. Top stuff.

But almost everytime I use Sumatra the Tab Labels Dissapear.

I took 3 screen shots, 1 when it was working, another when it stopped working, and a 3rd hovering over the tab label.

Ive gotten the label to break twice with just one pdf by just mousing over the tab label, but I can reproduce the issue by opening and closing multiple pdfs usually mousing over the label breaks it if its not broken on load / close.

![label gone](https://github.com/sumatrapdfreader/sumatrapdf/assets/148913159/dcac135e-9c33-4e8a-8095-f40cee1f14be)
![label](https://github.com/sumatrapdfreader/sumatrapdf/assets/148913159/896c5791-7a77-4d53-9b77-dad2fd54c618)
![label hover over](https://github.com/sumatrapdfreader/sumatrapdf/assets/148913159/5e41257f-79ed-496e-9fe3-201288263656)

Still a fantastic update.

**Comment by kjk** [comment]
I can reproduce this, opened a bug https://github.com/sumatrapdfreader/sumatrapdf/issues/3799

**Comment by hrrrps** [comment]
Guess that means you don't need those screenshots anymore, right?

24 Oct 2023, 23:41 by ***@***.***:

**Reply by kjk** [reply]
yes, it's already been fixed

**Comment by hrrrps** [comment]
great, thank you

Oct 26, 2023, 00:03 by ***@***.***:
